index-singlecell.md

```diff
@@ -28,27 +26,35 @@ The server is part if the European Galaxy server and is maintained by the [RNA B
 {:toc}
 
 
-# Get started with
+# Get started with #singlecell
```

Comment:
Is the second `#` needed?

Reply:
Well it's an interesting point
The first # just makes it a title but the second one is to subtly ingrain into users heads the idea of tagging stuff with #singlecell so we eventually (dreaming longterm) end up with a wealth of useful workflows shared by users all appropriately tagged and thus searchable with #singlecell

Reply:
mh, ok, we merge quickly and you can iterate - its your site overall :)